xpallets/assets-registrar/src/verifier.rs

```diff
@@ -1,52 +1,53 @@
-use super::*;
+// Copyright 2019-2020 ChainX Project Authors. Licensed under GPL-3.0.
 
-pub const MAX_TOKEN_LEN: usize = 32;
-pub const MAX_DESC_LEN: usize = 128;
+use xpallet_protocol::{ASSET_DESC_MAX_LEN, ASSET_NAME_MAX_LEN, ASSET_SYMBOL_MAX_LEN};
 
-/// Visible ASCII char [0x20, 0x7E]
-#[inline]
-fn is_ascii_invisible(c: &u8) -> bool {
-    *c < 0x20 || *c > 0x7E
+use super::*;
+
+/// Token can only use ASCII alphanumeric character or "-.|~".
+pub fn is_valid_token<T: Trait>(token: &[u8]) -> DispatchResult {
```

Comment:
```suggestion
pub fn is_valid_asset_symbol<T: Trait>(asset_symbol: &[u8]) -> DispatchResult {
```

Since you are using `ASSET_SYMBOL_MAX_LEN`, better to also align the function name.